resources/schema/spidacalc/results/component.schema

```diff
@@ -41,44 +41,132 @@
+        }, {
+          "description": "moment result fields",
+          "required": ["deflection", "applied", "actual", "internal"]
+        }, {
+          "description": "error result fields",
+          "required": ["deflection"]
+        }],
+        "properties": {
+          "height": {
+            "$ref": "../../general/measurable.schema"
+          },
+          "stress": {
+            "$ref": "../../general/measurable.schema"
+          },
+          "actual": {
+            "type": "number"
+          },
+          "status": {
+            "$ref": "../../spidacalc/results/enums/status.schema"
+          },
+          "components": {
+            "type": "array"
+          },
+          "applied": {
+            "description": "any externally applied forces at this height",
+            "$ref": "../../general/point_load.schema"
+          },
+          "internal": {
+            "description": "internal forces at this height",
+            "$ref": "../../general/point_load.schema"
+          },
+          "deflection": {
+            "description": "the deflection from the starting position of the pole at this height",
+            "$ref": "../../general/triad.schema"
+          },
+          "momentOfInertia": {
+            "$ref": "../../general/measurable.schema"
+          },
+          "crossArea": {
+            "$ref": "../../general/measurable.schema"
+          },
+          "sectionModulus": {
+            "$ref": "../../general/measurable.schema"
           }
         }
       }
     },
     "allowable": {
-      "description":"allowable S.F. or percent",
+      "description": "allowable S.F. or percent",
       "type": "number"
     },
     "allowableEngineering": {
-      "description":"allowable engineering value.  This is usually pulled from the material properties i.e. 8000 psi for wood.",
+      "description": "allowable engineering value.  This is usually pulled from the material properties i.e. 8000 psi for wood.",
       "$ref": "../../general/measurable.schema"
     },
     "ratio": {
-      "description":"the ratio between actual and allowable",
+      "description": "the ratio between actual and allowable",
       "type": "number"
     },
     "windDirection": {
-      "description":"the direction in compass degress the wind was blowing for these results",
+      "description": "the direction in compass degress the wind was blowing for these results",
       "type": "integer"
     },
     "failingRatio": {
-      "description":"the ratio at which this analysis will fail",
+      "description": "the ratio at which this analysis will fail",
       "type": "number"
     },
-    "message":{
-      "description":"an optional message with additional information about these results",
+    "message": {
+      "description": "an optional message with additional information about these results",
       "type": "string"
     },
     "additionalValues": {
```

Comment:
Additional values is now an array of objects that have values and tags.  Tags are informational on how to display the line.